packages/compass-indexes/src/components/property-column/property-column.jsx

```diff
@@ -2,19 +2,37 @@ import map from 'lodash.map';
 import { format } from 'util';
 import React, { PureComponent } from 'react';
 import PropTypes from 'prop-types';
-import ReactTooltip from 'react-tooltip';
-import { InfoSprinkle } from 'hadron-react-components';
 import { shell } from 'electron';
 import getIndexHelpLink from '../../utils/index-link-helper';
 
-import classnames from 'classnames';
-import styles from './property-column.module.less';
+import {
+  spacing,
+  css,
+  Tooltip,
+  Body,
+  Badge,
+  BadgeVariant,
+  Icon,
+} from '@mongodb-js/compass-components';
 
-const TOOLTIP_ID = 'index-property';
+const containerStyles = css({
+  width: 'auto',
+  paddingLeft: spacing[4],
+  paddingRight: spacing[4],
+  marginTop: spacing[1],
+  '*:not(:last-child)': {
+    marginRight: spacing[1],
+  },
```

Comment:
Targeting children like that is error prone and especially with css-in-js should be an exception rather than the rule. This should either be applied to children or done with a gap css property that is specifically a parent prop

Reply:
thank you! i really like the idea of gap! let me try that.